core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -307,6 +308,7 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
         final RequestHeaders headers = req.headers();
         final ProxiedAddresses proxiedAddresses = determineProxiedAddresses(channel, headers);
         final InetAddress clientAddress = config.clientAddressMapper().apply(proxiedAddresses).getAddress();
+        final EventLoop eventLoop = channel.eventLoop();
```

Comment:
Let's rename it to `channelEventLoop` because we now have another `serviceEventLoop`.